addons/binding/org.openhab.binding.lgwebos/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lgwebos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="WebOSTV">
+		<label>WebOS TV</label>
+		<description>WebOS based smart TV</description>
+
+		<channels>
+			<channel id="power" typeId="commandType">
+				<label>Power</label>
+				<description>Via this binding TV can only be powered off, not on.</description>
+			</channel>
+			<channel id="mute" typeId="commandType">
+				<label>Mute</label>
+				<description>Current Mute Setting</description>
+			</channel>
+			<channel id="volume" typeId="volumeChannelType" />
+			<channel id="channel" typeId="stringType">
+				<label>Channel Number</label>
+				<description>Current Channel Setting</description>
+			</channel>
+			<channel id="channelUp" typeId="commandType">
+				<label>Channel Up</label>
+				<description>One Channel Up</description>
+			</channel>
+			<channel id="channelDown" typeId="commandType">
+				<label>Channel Down</label>
+				<description>One Channel Down</description>
+			</channel>
+			<channel id="channelName" typeId="channelNameChannelType" />
+			<channel id="toast" typeId="stringType">
+				<label>Toast</label>
+				<description>Send a message onto the TV screen.</description>
+			</channel>
+			<channel id="mediaPlayer" typeId="mediaPlayerType" />
+			<channel id="mediaStop" typeId="commandType">
```

Comment:
Can't this be removed as mediaPlayer already supports a "stop" command?

Reply:
I see a PlayPauseType wich is mapped to play and pause on webos, but no equivalent stop type.

Reply:
@kaikreuzer I agree with @sprehn . We use a separate channel for `stop` in the Kodi binding as well. I am not aware of the Player item to be capable of accepting something like a "stop" command.